src/wovnio/utils/hash_gen.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Was it a mistake to upload this file? It seems ununsed.

Reply:
Nope. This is a command line utility to generate the hash needed to login. If the customers need to login themselves they need this tool as well. More details on how usage can be found [here](https://wovnio.atlassian.net/browse/ST-277?focusedCommentId=17815).

Reply:
I understand now, thanks. Is it gonna be documented somewhere else?

Reply:
Because this is considered internal operation procedures, I'm thinking adding a section on the Git Wiki in one of our private repositories, or a new confluence page.